awx/ui/client/lib/components/layout/side-nav.directive.js

```diff
@@ -4,14 +4,24 @@ function atSideNavLink (scope, element, attrs, ctrl) {
     scope.layoutVm = ctrl;
 }
 
-function AtSideNavController () {
+function AtSideNavController ($scope) {
     let vm = this || {};
 
     vm.isExpanded = false;
 
     vm.toggleExpansion = () => {
         vm.isExpanded = !vm.isExpanded;
     }
+
+    document.body.onclick = (e) => {
```

Comment:
I feel like this should go in the directive, if possible (as it is DOM manipulation/event binding)